app.config.ts

```diff
@@ -11,5 +11,10 @@ export default ({config}: ConfigContext): Partial<ExpoConfig> => {
   config.android!.config!.googleMaps!.apiKey = process.env.ANDROID_GOOGLE_MAPS_API_KEY;
   config.extra!.log_level = process.env.LOG_LEVEL != null ? process.env.LOG_LEVEL : 'info';
 
+  if (config.plugins === undefined) {
```

Comment:
@beaucollins when I updated to Expo SDK 50 the console told me to add this here - wanted to check if you had console scroll-back to confirm?

Reply:
So with a completely clean install of dependencies and running `yarn run start` I don't see anything related to this in either the `expo start` process or the logs in the app.

Reply:
Ah sorry it was on `yarn add expo@50` - since it can't auto-parse and auto-update `.ts` configs. No worries.